Documentation/ColumnsConfig/Type/Category/Properties/Index.rst

```diff
@@ -13,12 +13,13 @@ Special properties
 
```

Comment:
Maybe use a glob, if all properties should be listed alphabetically

Reply:
Ack